coffee_cmd/src/coffee_term/command_show.rs

```diff
@@ -60,14 +60,14 @@ pub fn show_remote_list(remote_list: Result<CoffeeRemote, CoffeeError>) -> Resul
     table.divider();
 
     for repository in &repositories {
-        let commit_id = match &repository.commit_id {
-            Some(commit_id) => commit_id.to_owned(),
-            None => String::from(""),
-        };
-        let date = match &repository.date {
-            Some(date) => date.to_owned(),
-            None => String::from(""),
-        };
+        let commit_id = repository
```

Comment:
doing all in one line is good sometimes, but when the chain of call starts to be a little bit too long, it is better to to split the line. Maybe each time you call `unwrap` you will split a line?

Reply:
I agree
looks cleaner now